cocassel/milk-buddy
# Update database with container information

When a user selects their containers on the container selection page, the container table on the database is updated to including the size and amount remaining

**Testing Instructions**
On the container page, increase the number of containers and click on the "Next" button to save the container information to the database. Save the dairy.db from the Device File Explorer locally. Open the DB browser and verify that the container table has been updated.
